src/LifeCycleHistory.js

```diff
@@ -105,6 +106,19 @@ export default class LifecycleHistory {
         if (event.data.query != undefined) {
           const intentReader = new IntentReader();
           const query = JSON.parse(event.data.query);
+
```

Comment:
change 
 // Establishing a pubSub connection when an intent is received in the navigateTo intent with the following parameters. 
to 
 // Establishing a pubSub connection if FCA receives an intent in the navigateTo event with the following parameters.

Reply:
addressed the comments